stack.yaml

```diff
@@ -1,18 +1,11 @@
-resolver: lts-17.15
+resolver: nightly-2022-06-10 # GHC 9.2.3
```

Comment:
It's unusual to make the primary way to build Stack a nightly image, but I don't have a huge issue. @borsboom any thoughts?

Reply:
At the end of the day it's a snapshot and if it [the dependencies it needs] builds it builds, right? I felt this way too until recently, but the moment we get this patch landed I'm going to take my system to GHC 9.2 and if that means `nightly` for now so be it. I'll return us to `stable` ^W `lts` once we've got one, I imagine. Same would apply here?